extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -455,35 +447,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return { notebookPaths: notebookPaths, bookPaths: bookPaths };
 	}
 
-	private runThrottledAction(resource: string, action: () => void) {
```

Comment:
Do we know why this was thought to be necessary to start with?

Reply:
No, it was there from when I took over. @chlafreniere  do you have any context here?

Reply:
https://github.com/microsoft/azuredatastudio/pull/6420

Reply:
There's some description in the PR. Please check if that behavior still exists, where multiple calls are coming in.